ui/src/duck/designation.ts

```diff
@@ -0,0 +1,42 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { Designation, fetch } from 'src/entities/Designation';
```

Comment:
Let's rename to `fetch` to `fetchAll`. `fetch` being a global in browser might trip someone